### DISCUSSION TITLE
Variable name fixup

### DISCUSSION THREAD

**Discussion by Dogers** [Kong/insomnia#3688] [Ideas]
- Insomnia Version: 5.16.6
- Operating System: Win10 x64

## Details

An issue I've just come across - I've been working on creating a bunch of requests for some software and then found someone's already created a chunk more, albeit in Postman format. I've imported them, but obviously they've used different variable names to me, for example they've used SERVER_NAME but I'm using SERVER_ADDRESS. The problem now is I have to visit each request and replace each undeclared variable they've used with my version!

So what I'd like to see is a way to see a list of all (or at least invalid?) variables/tags in the workspace, with an option to swap them for another across all requests. The current way round this is to export, find & replace on the json and then reimport, but this is a PITA :)

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by gschier** [comment]
This seems like a fairly rare use-case, but I'd be open to someone submitting a PR for this if someone wanted to take it on. I envision a button in the environment dialog to trigger a prompt to rename a variable. It could provide a list of current variable on the left, with a text-box on the right to enter a new value.

Note: Another idea is to implement a more generic find and replace, which could satisfy more use-cases.
